cli/cli.go

```diff
@@ -173,10 +176,24 @@ func main() {
 			os.Exit(6)
 		}
 
-		b, err := json.MarshalIndent(torrents, "", "\t")
+		je := json.NewEncoder(os.Stdout)
+		je.SetIndent("", "\t")
+		if err := je.Encode(torrents); err != nil {
+			os.Exit(7)
+		}
+	}
+
+	if listAccounts {
+		accounts, err := deluge.KnownAccounts()
 		if err != nil {
+			fmt.Fprintf(os.Stderr, "ERROR: could not list all accounts: %v\n", err)
+			os.Exit(6)
+		}
+
+		je := json.NewEncoder(os.Stdout)
+		je.SetIndent("", "\t")
+		if err := je.Encode(accounts); err != nil {
 			os.Exit(7)
 		}
-		fmt.Println(string(b))
```

Comment:
streaming the output, instead of writing it into a buffer and printing all at once.